src/main/java/org/broadinstitute/hellbender/utils/read/GATKRead.java

```diff
@@ -307,6 +304,20 @@ default String getBasesString() {
      */
     byte[] getBaseQualities();
 
+    /**
+     * If the original base quality scores have been store in the "OQ" tag will return the numeric
+     * score as a byte[]
+     */
+    default byte[] getOriginalBaseQualities() {
```

Comment:
It doesn't seem worth including this as a first class function.  It already exists in ReadUtils.